spring-cloud-starter-stream-source-rabbit/README.adoc

```diff
@@ -43,19 +43,23 @@ The **$$rabbit$$** $$source$$ has the following options:
 //tag::configuration-properties[]
 $$rabbit.enable-retry$$:: $$true to enable retry.$$ *($$Boolean$$, default: `$$false$$`)*
 $$rabbit.initial-retry-interval$$:: $$Initial retry interval when retry is enabled.$$ *($$Integer$$, default: `$$1000$$`)*
-$$rabbit.mapped-request-headers$$:: $$Headers that will be mapped.$$ *($$String[]$$, default: `$$[STANDARD_REQUEST_HEADERS]$$`)*
+$$rabbit.mapped-request-headers$$:: $$Headers that will be mapped.$$ *($$String)[]$$, default: `$$[STANDARD_REQUEST_HEADERS]$$`)*
```

Comment:
Hm... Something wrong with the type parser when the story is about an array...
Or Maven Docs pluging or Spring Boot metadata generator...

Reply:
is this something to address in this PR or should an issue be made against another project?

Reply:
No, that's not your fault. I'm working with Stephane Nicoll on this issue.
Stay tuned!

Reply:
OK! As you see the Boot issue has been fixed: https://github.com/spring-projects/spring-boot/issues/11512.

I don't think that we have time to wait for the next Boot Milestone, so let's merge it as is.
And we can come back for README regeneration when an appropriate parent version is available for us: https://github.com/spring-cloud-stream-app-starters/app-starters-release/issues/132

Reply:
ok works for me 👍